akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -427,6 +497,14 @@ trait FlowOps[+Out, +Mat] {
    *
    * `n` must be positive, and `d` must be greater than 0 seconds, otherwise
    * IllegalArgumentException is thrown.
+   *
+   * '''Emits when''' the configured time elapses since the last group has been emitted
+   *
+   * '''Backpressures when''' never
```

Comment:
sure? shouldn’t it be «a group has been assembled and downstream backpressures»?